wrt/wrt-internetstdmanu-android-tests/internetstdmanu/Crosswalk_WebApp_HttpAuth_Failure.html

```diff
@@ -41,8 +41,8 @@
       <strong>Test steps:</strong>
     </p>
     <ol>
-      <li>Package a web app using the package command, for example:"python make_apk.py --package=com.httpauthentication --name=HttpAuthentication --app-url=http://httpbin.org/basic-auth/user/passwd" </li>
-      <li>Install the apk via command "adb install Httpauthentication.apk"</li>
+      <li>Package a web app using the package command, for example:"crosswalk-pkg -m '{"name": "HttpAuthentication", "xwalk_package_id": "org.xwalk.httpauthentication","start_url": "http://httpbin.org/basic-auth/user/passwd"}'" -t x86 -c /path/to/crosswalk**.zip /path/to/apk_saved_directory</li>
+      <li>Install the apk via command "adb install org.xwalk.httpauthentication*.apk"</li>
```

Comment:
No need to provide the command of app install. The apk name maybe updated as the last portion of package id, see XWALK-5658.